_posts/2018-06-03-soapbox-science-feedback.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title:  "The experience of Soapbox Science Cardiff 2018"
+date:   2018-05-24
+comments: False
+---
+
+
+[Soapbox Science](http://soapboxscience.org/) 2018 took place in Cardiff on the
+\\(2^{nd}\\) of June. The aim of the event is to promote STEM sciences
+and more specifically the role of women. This year the event was located outside
+Cardiff Central Library.
+
+<p align="center">
+  <img src="{{site.baseurl}}/assets/images/soap_box_2018.jpg" style='height: 60%; width: 80%; object-fit: contain'>
+</p>
```

Comment:
Maybe link out to the SS_Cardiff twitter account and/or their blog post? (I think I saw them post a blog post on twitter, I might be wrong)